<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinSplitDialogBase</class>
 <widget class="QDialog" name="JoinSplitDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JoinSplit</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="GridLayerCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="JoinTableCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>64</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>64</y>
     <width>64</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Join table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>64</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Field</string>
   </property>
  </widget>
  <widget class="QToolButton" name="OutputButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Folder</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OutputLine">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>190</y>
     <width>231</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="JoinFieldCombo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>JoinSplitDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>JoinSplitDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
